### ISSUE TITLE
Keyboard combination Ctrl+Shift+. doesn't work reliably in Firefox

### ISSUE THREAD

**Issue by martinvonwittich** [zammad/zammad#3644]
### Infos:

* Used Zammad version: 4.0.x
* Installation method (source, package, ..): docker-compose
* Operating system: Debian 10
* Database + version: from docker-compose
* Elasticsearch version: from docker-compose
* Browser + version: Firefox 89.0.2

### Expected behavior:

* Pressing the keyboard combination Ctrl+Shift+. to copy the ticket ID should would reliably.

### Actual behavior:

* Pressing Ctrl+Shift+. sometimes doesn't do anything at all. As there is no visual feedback when a ticket ID has been copied, this is only noticed when actually pasting the clipboard contents and realizing that it's not the expected ticket ID.

### Steps to reproduce the behavior:

* Press Ctrl+Shift+. in Firefox to copy the ID of a Zammad ticket. The problem isn't 100% reproducible - sometimes it works, sometimes it doesn't. It should be pretty obvious though when using Zammad productively every day and using Ctrl+Shfit+. regularly.

As far as I can remember, I never had this issue with Chrome/Chromium, therefore I believe it only occurs in Firefox.

On Linux, [bash - Detect clipboard copy/paste event and modify clipboard contents - Ask Ubuntu [askubuntu.com]](https://askubuntu.com/questions/1167026/detect-clipboard-copy-paste-event-and-modify-clipboard-contents) can be helpful to monitor the clipboard for changes to easily tell if the keyboard shortcut worked:

```
martin@martin ~/clipnotify % cat test.sh 
#!/bin/sh

while ./clipnotify
do
  xsel -b | grep -v 'No newline at end of selection'
done
```

When running this script, and then switching between two opened tickets and pressing Ctrl+Shift+. after each switch, I can reproduce the problem reliably with exactly 4 attempts:

1. The first attempts works and copies the ID of the current ticket.
2. The second attempt works and copies the ID and title of the current ticket.
3. The third attempt works and copies ID, title and URL of the ticket.
4. The fourth and all further attempts don't do anything at all.

No JavaScript exception or anything other helpful is logged to the console.

Yes I'm sure this is a bug and no feature request or a general question.

**Comment by MrGeneration** [comment]
Do you happen to know if this is OS specific?

**Comment by martinvonwittich** [comment]
> Do you happen to know if this is OS specific?

Hard to say. I booted into Windows 10 and tried my approach to reproduce it that I've illustrated above, by alternating between two opened tickets, and pressing Ctrl+Shift+. each time:

```
IServ#962638437
IServ#962642781: Title A
IServ#962638437: Title B
https://zammad.ourcompany.tld/#ticket/zoom/264118
IServ#962638437: Title B
https://zammad.ourcompany.tld/#ticket/zoom/264118
IServ#962638437: Title B
https://zammad.ourcompany.tld/#ticket/zoom/264118
IServ#962638437: Title B
https://zammad.ourcompany.tld/#ticket/zoom/264118
```

The expected output would have been an alternating ticket ID each time (sometimes with title and/or with link; that Zammad confuses Ctrl+Shift+. pressed every second with Ctrl+Shift+.. or Ctrl+Shift+... is understandable), but at some point it just stops updating the clipboard, and the last result (ID, title and URL of ticket B) will remain.

This would indicate that Firefox on Windows 10 is also affected, but unfortunately, I can also reproduce it with this approach in Chrome on Windows 10. This might just indicate that my reproduction approach is flawed :/

As I don't usually work on Windows, I don't have any "real life" experience with this issue on Windows. A colleague that I've asked who works on Firefox + Windows has said that he also sometimes feels as if Ctrl+Shift+. doesn't work reliably, but that is of course not really confident.
